src/plots/plots.js

```diff
@@ -1039,9 +1039,21 @@ plots.supplyTransformDefaults = function(traceIn, traceOut, layout) {
             _module = transformsRegistry[type],
             transformOut;
 
+        /*
+         * Supply defaults may run twice. First pass runs all supply defaults steps
```

Comment:
Questions:

- I assume this refers to the way `supplyTraceDefaults` runs [on traces and then again on expanded traces](https://github.com/plotly/plotly.js/blob/master/src/plots/plots.js#L799-L812)?
- Which `_module` is added? I think that sometimes refers to the trace module and sometimes to the transform module (and every now and then to the base plot module, though usually it's called `baseModule` or something, I think)

Reply:
Answer to second: the *transform* module.

Reply:
Yep that is what it is referring to